shared.bazelrc

```diff
@@ -320,6 +321,7 @@ common:linux-workflows --config=target-linux-x86
 common:linux-workflows --config=workflows
 common:linux-workflows --config=buildbuddy_remote_executor
 common:linux-workflows --build_metadata=TAGS=linux-workflow
+common:linux-workflows --remote_header=x-buildbuddy-platform.enable-vfs=true
```

Comment:
maybe worth trying a run with `--remote_instance_name=foo` - I think these props aren't part of the AC key so some of the passing tests might be cached results

Reply:
I'll leave it as is for now since we can't force things to run on bare metal machines.